ocs_ci/deployment/deployment.py

```diff
@@ -587,11 +588,23 @@ def deploy_ocs_via_operator(self, image=None):
             templating.dump_data_to_temp_yaml(multus_data, multus_data_yaml.name)
             run_cmd(f"oc create -f {multus_data_yaml.name}")
 
+        disable_addon = config.DEPLOYMENT.get("ibmcloud_disable_addon")
+
         if config.ENV_DATA["platform"] == constants.IBMCLOUD_PLATFORM:
             ibmcloud.add_deployment_dependencies()
             if not live_deployment:
                 create_ocs_secret(self.namespace)
-        self.subscribe_ocs()
+        if (
+            config.ENV_DATA["platform"] == constants.IBMCLOUD_PLATFORM
+            and live_deployment
+            and not disable_addon
+        ):
+            self.deploy_odf_addon()
+            return
```

Comment:
why do we skip installation if users choose to enable addon for IBM platform?

Reply:
The installation is handled fully by addon so no need to do any other call from this function hence Gangadhar is returning from here. Then basically we are basically continue with the rest of the verification we call after this deploy_ocs_via_operator function.